src/gtc/dace/nodes.py

```diff
@@ -40,32 +40,58 @@ def _set_expansion_order(
     node._expansion_specification = res
 
 
+class PickledProperty:
+    def to_json(self, obj):
+        protocol = pickle.DEFAULT_PROTOCOL
+        pbytes = pickle.dumps(obj, protocol=protocol)
+        jsonobj = dict(pickle=base64.b64encode(pbytes).decode("utf-8"))
+        return jsonobj
+
+    @classmethod
+    def from_json(cls, d, sdfg=None):
+        b64string = d["pickle"]
+        byte_repr = base64.b64decode(b64string)
+        return pickle.loads(byte_repr)
```

Comment:
Probaly for manual testing, but in case I wanted to call out that from a quick search in gt4py, these methods _seem_ to be neither used nor tested.